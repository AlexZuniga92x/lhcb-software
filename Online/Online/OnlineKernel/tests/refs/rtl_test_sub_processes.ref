Parent: Started process:SLEEPER_1 output redirected to:/dev/null
Parent: Started process:SLEEPER_2
Parent: Started process:SLEEPER_3 output redirected to:/dev/null
Parent: Started process:SLEEPER_4
Parent: Started process:SLEEPER_5 output redirected to:/dev/null
Parent: Started process:SLEEPER_6
Child: Starting process:SLEEPER_2
Child: Starting process:SLEEPER_4
Child: Starting process:SLEEPER_6
Stop process SLEEPER_2
SLEEPER_2    Process starting...
SLEEPER_2    arg0:rtl_test_process_sleep
SLEEPER_2    arg1:3
SLEEPER_2    Process sleeping for 3 seconds.
RTL:Handled signal: 15 [SIGTERM] Old action:(nil)
Stop process SLEEPER_4
SLEEPER_4    Process starting...
SLEEPER_4    arg0:rtl_test_process_sleep
SLEEPER_4    arg1:5
SLEEPER_4    Process sleeping for 5 seconds.
RTL:Handled signal: 15 [SIGTERM] Old action:(nil)
Stop process SLEEPER_6
SLEEPER_6    Process starting...
SLEEPER_6    arg0:rtl_test_process_sleep
SLEEPER_6    arg1:7
SLEEPER_6    Process sleeping for 7 seconds.
RTL:Handled signal: 15 [SIGTERM] Old action:(nil)
Process SLEEPER_2 ended. Status=0
Process SLEEPER_4 ended. Status=0
Process SLEEPER_6 ended. Status=0
Process SLEEPER_1 ended. Status=0 output redirected to:/dev/null
Process SLEEPER_3 ended. Status=0 output redirected to:/dev/null
Process SLEEPER_5 ended. Status=0 output redirected to:/dev/null
All processes ended
