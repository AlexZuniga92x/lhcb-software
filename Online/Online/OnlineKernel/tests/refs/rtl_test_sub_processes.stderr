[FATAL] Process: '' (ExitSignalHandler) RTL:Handled signal: 15 [SIGTERM] Old action:(nil) Mem:0x264900000bce Code:00000000
[FATAL] Process: '' (ExitSignalHandler) RTL:Handled signal: 15 [SIGTERM] Old action:(nil) Mem:0x264900000bce Code:00000000
[FATAL] Process: '' (ExitSignalHandler) RTL:Handled signal: 15 [SIGTERM] Old action:(nil) Mem:0x264900000bce Code:00000000
